modules/dxp/apps/search-experiences/search-experiences-service/src/main/java/com/liferay/search/experiences/internal/search/spi/model/index/contributor/JournalArticleTextEmbeddingModelDocumentContributor.java

```diff
@@ -46,6 +51,21 @@
 
 	@Override
 	public void contribute(Document document, JournalArticle journalArticle) {
+		try {
+			if (!_journalArticleLocalService.isLatestVersion(
+					journalArticle.getGroupId(), journalArticle.getArticleId(),
+					journalArticle.getVersion(),
+					WorkflowConstants.STATUS_APPROVED)) {
+
+				return;
+			}
+		}
+		catch (PortalException portalException) {
+			_log.error(portalException);
+
+			return;
+		}
+
```

Comment:
Just checking: does it do the same logic that `document.addKeyword("head", JournalUtil.isHead(journalArticle));` does?

Reply:
@peerkar :point_up: